Documentation/Sprint 1/Sprint reflection.tex

```diff
@@ -0,0 +1,258 @@
+\usepackage[none]{hyphenat}
+\usepackage[parfill]{parskip}
+\usepackage{pgfplots}
+\usepackage[top=2 cm, bottom= 2 cm, left=1 cm, right=1 cm]{geometry}
+\usepackage{enumitem}
+\begin{document}
+
+
+\section*{Sprint Plan 1}
+Context : Multimedia services\\
+Group : Multimedia services 1\\
+
+
+
+\begin{table}[h]
+\begin{tabular}{|p{5cm}|p{1.0cm}|p{2.5cm}|p{1.8cm}|p{1.8cm}|p{1.0cm}|p{12cm}}
+\textbf{User Story} & \textbf{Task} & \textbf{Task assigned to} & \textbf{Estimated Effort per Task (Hours)} & \textbf{Actual Effort per Task (Hours)} & \textbf{Done} & \textbf{Notes}\\
+As a User, I want to be able to log into MoodCat.me & \begin{enumerate}[leftmargin=0.3cm,itemindent=0.1cm]
+\item 
+\item
+\item 
+\item 
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Gijs 
+\item[] Gijs 
+\item[] Jan-Willem 
+\item[] Jaap  
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 4 
+\item[] 6 
+\item[] 4 
+\item[] 6 
+\end{enumerate}
+&
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 4 
+\item[] 6 
+\item[] 2 
+\item[] 8 
+\end{enumerate}
+&
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Yes
+\item[] Yes
+\item[] Yes
+\item[] Yes
+\end{enumerate}
+&
+\begin{enumerate}[leftmargin=0.3cm,itemindent=0.1cm]
+\item[]
+\item[]
+\item[]
+\item[]
+\end{enumerate}
+ \\
+
+
+As a User, I want to be able to play music on MoodCat.me & \begin{enumerate}[leftmargin=0.3cm,itemindent=0.1cm]
+\item 
+\item 
+\item
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Eva 
+\item[] Gijs 
+\item[] Jaap 
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 2 
+\item[] 13 
+\item[] 8 
+\end{enumerate} 
+ & \begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 1 
+\item[] 12 
+\item[] 6 
+\end{enumerate}
+
+ & \begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Yes
+\item[] Yes
+\item[] Yes
+\end{enumerate}
+
+ & \begin{enumerate}[leftmargin=0.5cm,itemindent=0.1cm]
+\item Jan-Willem took over this issue.
+\item 
+\item Gijs took over this issue.
+\end{enumerate}
+
+\\
+
+As a Developer, I want to be able to run Algorithms on a server-side platform & \begin{enumerate}[leftmargin=0.3cm,itemindent=0.1cm]
+\item 
+\item
+\item 
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Eva 
+\item[] Tim
+\item[] Eva 
+\end{enumerate}
+
+& 
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 3 
+\item[] 3 
+\item[] 10 
+\end{enumerate}
+
+&
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] 3 
+\item[] 4 
+\item[]
+\end{enumerate}
+
+&
+\begin{enumerate}[leftmargin=0.1cm,itemindent=0.1cm]
+\item[] Yes
+\item[] Yes
+\item[] No
+\end{enumerate}
+
+&
+\begin{enumerate}[leftmargin=0.5cm,itemindent=0.1cm]
+\item Was a team effort with Eva as lead.
+\item Originally chosen for Jersey, but decided to use RESTeasy.
```

Comment:
While this is true, the it's only a different implementation of the `JAX-RS WS` standard interfaces. For us, there is no difference (only that we had a boilerplate for RESTeasy which kickstarted us). Furthermore, we also made the design decisions for Jackson (JSON parsing and encoding), Hibernate (object/relational mapping) and Guice (dependency injection).